frontend/src/stores/levels/actions.ts

```diff
@@ -13,4 +13,26 @@ async function getLevelsByRoleId(roleId: number): Promise<Level[] | undefined> {
   return role.levels
 }
 
-export { getLevelsByRoleId }
+async function getScoreResponse(
```

Comment:
These functions (and all score related functions) should be in another state management, not this. src/stores/quiz, you can create a new structure here